src/Schema/SnapshotStoreSchema.php

```diff
@@ -32,13 +32,17 @@ public static function create(Schema $schema, $snapshotName = 'snapshot')
     {
         $snapshot = $schema->createTable($snapshotName);
 
+        // UUID4 of linked aggregate
+        $snapshot->addColumn('aggregate_id', 'string', ['fixed' => true, 'length' => 36]);
+        // Class of the linked aggregate
         $snapshot->addColumn('aggregate_type', 'string');
```

Comment:
The aggregate_type column should have the same length from the [event store](https://github.com/prooph/event-store-doctrine-adapter/pull/30).